pandas/tests/apply/test_frame_apply.py

```diff
@@ -38,62 +38,65 @@ def int_frame_const_col():
 def test_apply(float_frame):
     with np.errstate(all="ignore"):
         # ufunc
-        applied = float_frame.apply(np.sqrt)
-        tm.assert_series_equal(np.sqrt(float_frame["A"]), applied["A"])
+        result = np.sqrt(float_frame["A"])
+        expected = float_frame.apply(np.sqrt)["A"]
+        tm.assert_series_equal(result, expected)
 
         # aggregator
-        applied = float_frame.apply(np.mean)
-        assert applied["A"] == np.mean(float_frame["A"])
+        result = float_frame.apply(np.mean)["A"]
+        expected = np.mean(float_frame["A"])
+        assert result == expected
 
         d = float_frame.index[0]
-        applied = float_frame.apply(np.mean, axis=1)
-        assert applied[d] == np.mean(float_frame.xs(d))
-        assert applied.index is float_frame.index  # want this
+        result = float_frame.apply(np.mean, axis=1)
+        expected = np.mean(float_frame.xs(d))
+        assert result[d] == expected
+        assert result.index is float_frame.index
+
+    # GH 9573
+    df = DataFrame({"c0": ["A", "A", "B", "B"], "c1": ["C", "C", "D", "D"]})
+    result = df.apply(lambda ts: ts.astype("category"))
```

Comment:
Maybe better keep this closer to the assertion, line 65?
Otherwise it is divided by the invalid axis test with its re-definition of ``df``.

Reply:
Thanks for catching this! I was moving it to a different file before I realized it should be done in a follow-up and put it back in the wrong place.